components/ModelLicense/licenseUtils.ts

```diff
@@ -0,0 +1,175 @@
+    default:
+      return status === 'allow' ? 'OK' : 'NG';
+  }
+};
+
+/**
+ * VRoid Hubで設定出来るVRM0のライセンスからテキストの表示色の種類を返す
+ * @param license
+ * @param property
+ * @returns
+ */
+export const getHubVrm0LicenseVariant = (
+  license: CharacterModelLicenseSerializer,
+  property: LicenseProperty,
+): 'default' | 'allow' | 'bold' => {
+  const status = license[property];
+
+  switch (property) {
+    case 'characterization_allowed_user':
+      return status === 'everyone' ? 'allow' : 'default';
+    case 'personal_commercial_use':
+      return status === 'disallow' || status === 'default' ? 'default' : 'allow';
+    case 'credit':
+      return status === 'necessary' ? 'bold' : 'default';
+    case 'modification':
+      return status === 'allow' ? 'allow' : 'default';
+    case 'redistribution':
+      return status === 'allow' ? 'allow' : 'default';
+    case 'sexual_expression':
+      return status === 'allow' ? 'allow' : 'default';
+    case 'violent_expression':
+      return status === 'allow' ? 'allow' : 'default';
+    case 'corporate_commercial_use':
+      return status === 'allow' ? 'allow' : 'default';
+    default:
+      return status === 'allow' ? 'allow' : 'default';
+  }
+};
+
+type vrm1MetaProperty = keyof VRM1Meta | 'personalCommercialUse' | 'modificationRedistribution';
+
+/**
+ * VRM1.0のライセンスから表示テキストを返す
+ * @param vrm1Meta
+ * @param property
+ * @returns
+ */
+export const getVrm1LicenseValue = (vrm1Meta: VRM1Meta, property: vrm1MetaProperty) => {
+  //「 個人の商用利用」は法人利用とプロパティが共用のため、commercialUsageの値を参照し判定する
+  if (property === 'personalCommercialUse') {
+    const status = vrm1Meta.commercialUsage;
+    if (typeof status === 'undefined') return '未設定';
+
+    return status === 'corporation' || status === 'personalProfit' ? 'OK' : 'NG';
+  }
+
+  // 「改変物の再配布」は改変とプロパティが共用のため、modificationの値を参照し判定する
+  if (property === 'modificationRedistribution') {
+    const status = vrm1Meta.modification;
+    if (typeof status === 'undefined') return '未設定';
+
+    return status === 'allowModificationRedistribution' ? 'OK' : 'NG';
+  }
+
+  const status = vrm1Meta[property];
+  if (typeof status === 'undefined') return '未設定';
+
+  switch (property) {
+    case 'avatarPermission':
+      return status === 'everyone' ? 'OK' : 'NG';
+    case 'allowExcessivelyViolentUsage':
+      return status === true ? 'OK' : 'NG';
+    case 'allowExcessivelySexualUsage':
+      return status === true ? 'OK' : 'NG';
+    case 'allowPoliticalOrReligiousUsage':
+      return status === true ? 'OK' : 'NG';
+    case 'allowAntisocialOrHateUsage':
+      return status === true ? 'OK' : 'NG';
+    case 'commercialUsage':
+      return status === 'corporation' ? 'OK' : 'NG';
+    case 'allowRedistribution':
+      return status === true ? 'OK' : 'NG';
+    case 'modification':
+      return status === 'allowModification' || status === 'allowModificationRedistribution' ? 'OK' : 'NG';
+    case 'creditNotation':
+      return status === 'necessary' ? '必要' : '不要';
```

Comment:
ここは`'required'`です
参考: https://github.com/pixiv/three-vrm/blob/9a1814ade92e2617cb4b96e6210e15e45e2ff935/packages/three-vrm-core/src/meta/VRM1Meta.ts#L89

```suggestion
      return status === 'required' ? '必要' : '不要';
```

Reply:
見落としていました！修正します。

HubVRM0では"necessary" / "unnecessary" 、VRM1では"required"/ "unnecessary"ですね！

Reply:
はい！その通りです。

Reply:
https://github.com/pixiv/VRoidHub-API-Example/pull/7/commits/b010d48e05c639cf63ebc870470957bdd7a606f2 にて修正しました